tests/event_status/Tribe/Events/Event_Status/Views/Single_ViewTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Tribe\Events\Event_Status\Views;
+
+use Tribe\Events\Event_Status\Status_Labels;
+use Tribe\Test\PHPUnit\Traits\With_Post_Remapping;
+use Tribe\Test\Products\WPBrowser\Views\V2\HtmlPartialTestCase;
+
+class Single_ViewTest extends HtmlPartialTestCase {
+	use With_Post_Remapping;
+
+	protected $partial_path = 'event-status/single/event-statuses';
+
+	/**
+	 * @test
+	 */
+	public function it_should_render_single_notice_with_no_name_or_reason() {
+		$event = $this->get_mock_event( 'events/single/1.json' );
+		$this->assertMatchesSnapshot( $this->get_partial_html( [
+			'event' => $event,
+			'status_labels' => new Status_Labels(),
+		] ) );
+	}
+
+	/**
+	 * @test
+	 */
+	public function it_should_render_single_canceled_name_reason_and_html() {
```

Comment:
Should also have one for name, no reason (and vice-versa) - to catch display hiccups there.
Do we not need these for any of the other views or is single the only one that shows reason and has logic for it?

Reply:
Single is the only review with the reason. I added two tests for no reasons.